### PR TITLE
PUSH

### PR DESCRIPTION
**Pull request by NaysKutzu**
-> Update .bat and .sh file
-> Fix some formatting in the README
-> Update license to 2023

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ npm run add-dev YOUR_ID
 
  By Donating, You Will Help Me To Maintain This Project 
 
-<img src="https://cdn.discordapp.com/emojis/809085860632985630.png" width="15px" height="15px"> 》[Sponsor](https://github.com/sponsors/CorwinDev)
+<img src="https://cdn.discordapp.com/emojis/809085860632985630.png" width="15px" height="15px"> 》 Sponsor](https://github.com/sponsors/CorwinDev)
```
Comment (tippfehlr) [inline README.md:112]: you need to leave the `[`. Insert your space in front of it, if you want.

File: README.md [hunk 2]
```diff
@@ -57,17 +57,17 @@ If you like this repository, feel free to leave a star ⭐ to motivate me!
...
     <img src="https://cdn.discordapp.com/attachments/778665159316209748/1055832339328024666/207117434-d98356b1-bf19-418e-9e12-0ef83e0d9a21.png">
   </a>
 </p>
 
-## <img src="https://cdn.discordapp.com/emojis/1009754836314628146.gif" width="25px" height="25px">》Requirements
-- NodeJs v17+
-- Java v13 for lavalink server.
+## <img src="https://cdn.discordapp.com/emojis/1009754836314628146.gif" width="25px" height="25px">》 Requirements
+- Node.js v17+
+- Java v13 for lava link server.
```
Comment (tippfehlr) [inline README.md:70]: lavalink is one word.

File: start.bat [hunk 3]
```diff
@@ -1,2 +1,3 @@
+@echo off
 node . --trace-warnings
 PAUSE
```
Comment (tippfehlr) [inline start.bat:3]: It’s customary to end source files with a newline.

Have a look at [^1] for reasons.

[^1]: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

**Comment by tippfehlr** [comment]
Whoops you closed this, sorry :)

**Comment by NaysKutzu** [comment]
> Whoops you closed this, sorry :)

Yeah idk why I made this it was for a tool I was developing to fix spelling mistakes and more

**Pull request closed without merging**